packages/identifier-resolution/src/functions/externalIdentifierFunctions.ts

```diff
@@ -49,13 +57,15 @@ export async function resolveExternalIdentifier(
     return resolveExternalJwkIdentifier(opts, context)
   } else if (isExternalIdentifierCoseKeyOpts(opts)) {
     return resolveExternalCoseKeyIdentifier(opts, context)
+  } else if (isExternalIdentifierEntityIdOpts(opts)) {
+    return resolveExternalEntityIdIdentifier(opts, context)
```

Comment:
rename to OIDFEntityId please. The name entityId alone is a bit to generic. OID4VP should also not have called it like that

Reply:
fixed